lib/perl/Genome/SoftwareResult/ImportedFile.pm

```diff
@@ -0,0 +1,26 @@
+package Genome::SoftwareResult::ImportedFile;
+
+use strict;
+use warnings;
+use Genome;
+
+class Genome::SoftwareResult::ImportedFile {
+    is => 'Genome::SoftwareResult::StageableSimple::SingleFile',
+    has_input => [
+        source_file_path => {
+            is => 'Path',
+        }
+    ],
+};
+
+sub _run{
+    my $self = shift;
+    Genome::Sys->copy_file($self->source_file_path, $self->_temp_staging_file_path);
```

Comment:
Speaking of `Genome::Model::Tools::DetectVariants2::Result::Manual`, it uses the `_needs_symlinks_followed_when_syncing` feature of `SoftwareResult::Stageable` to avoid making two copies of the file.  That might be useful here, too.